src/Console/SecurityCommand.php

```diff
@@ -43,10 +29,14 @@ public function handle()
         // get the path to composer.lock
         $composerLock = base_path('composer.lock');
 
-        // and feed it into the SecurityChecker
-        $checkResult = json_decode((string)$this->checker->check($composerLock), true);
+        $parser = new AdvisoryParser((new AdvisoryFetcher)->fetchAdvisories());
```

Comment:
I usually avoid class instantiation in the class itself in favor of Dependency Injection through the constructor. This way, we can mock the dependency in the unit tests.